bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueBridgeDiscoveryParticipant.java

```diff
@@ -92,4 +113,20 @@
         }
         return null;
     }
+
+    @Override
+    public long getRemovalGracePeriodSeconds(RemoteDevice device) {
+        try {
+            Configuration conf = configAdmin.getConfiguration("binding.hue");
+            Dictionary<String, @Nullable Object> properties = conf.getProperties();
+            Object property = properties.get(HueBindingConstants.REMOVAL_GRACE_PERIOD);
+            if (property != null) {
+                removalGracePeriodSeconds = Integer.valueOf(property.toString()).longValue();
+            }
+        } catch (IOException | IllegalStateException | NullPointerException | NumberFormatException e) {
```

Comment:
What `NullPointerException` are you expecting in the code above?

Reply:
Removed.